### PR TITLE
C2LC-595: Move actions simplification to icon button quad.

### PR DESCRIPTION
**Pull request by the-t-in-rtf**
See [C2LC-595](https://issues.fluidproject.org/browse/C2LC-595) for details.

### CODE REVIEW

File: src/Themes.scss [hunk 1]
```diff
@@ -790,6 +790,12 @@ body.gray-theme {
         }
     }
 
+    // TODO: Need design input for these.
```
Comment (sbates-idrc) [inline src/Themes.scss]: What do we need here?

**Comment by the-t-in-rtf** [comment]
This was work in progress before I saw your comment in the chat on where to
find comparable styles to use there.  It's been removed now?

**Comment by sbates-idrc** [comment]
> This was work in progress before I saw your comment in the chat on where to find comparable styles to use there. It's been removed now?

I was looking at changes to this PR in the GitHub Notifications UI and somehow it was showing the `TODO: Need design input for these` comment as a diff, even though it had been already removed by a later commit. Once I saw that it had been removed, I deleted my comment.

Apologies for the noise.

**Comment by sbates-idrc** [comment]
In high-contrast, the new icon has filled-in squares, rather than outlines.

image

**Comment by sbates-idrc** [comment]
The style changes have also affected the disabled Delete All button in high-contrast.

With this PR:

image

Was before (develop-1.2 and public site):

image

**Comment by the-t-in-rtf** [comment]
> > This was work in progress before I saw your comment in the chat on where to find comparable styles to use there. It's been removed now?
> 
> I was looking at changes to this PR in the GitHub Notifications UI and somehow it was showing the `TODO: Need design input for these` comment as a diff, even though it had been already removed by a later commit. Once I saw that it had been removed, I deleted my comment.
> 
> Apologies for the noise.

No worries, I figured it was something like that.

**Comment by the-t-in-rtf** [comment]
@sbates-idrc, I fixed the two issues in the contrast theme, this should be ready for further review.

File: src/Themes.scss [hunk 2]
```diff
@@ -1528,12 +1539,14 @@ body.contrast-theme {
...
     }
 
-    .IconButton {
-        background-color: white;
-
+    .App__ActionsMenu.IconButton:not(.App__ActionsMenu__toggle-button) {
         svg path {
             fill: black;
         }
+    }
```
[the remark this answers is not in the document]
Comment (the-t-in-rtf) [reply src/Themes.scss]: Removed.

**Pull request merged**